src/main/java/org/broadinstitute/consent/http/models/support/SupportTicket.java

```diff
@@ -9,10 +13,37 @@ public class SupportTicket {
 
     private SupportRequest request;
 
-    public SupportTicket(SupportRequester requester, String subject, List<CustomRequestField> customFields, SupportRequestComment comment) {
-        this.request = new SupportRequest(requester, subject, customFields, comment);
+    /**
+     * Constructs a ticket with the proper structure to request support via Zendesk
+     *
+     * @param name        The name of the user requesting support
+     * @param type        The type of request ("question", "incident", "problem", "task")
+     * @param email       The email of the user requesting support
+     * @param subject     Subject line of the request
+     * @param description Description of the task or question
+     * @param url         The API url of this request
+     */
+    public SupportTicket(String name, SupportRequestType type, String email, String subject, String description, String url) {
+        if (Objects.isNull(name) || Objects.isNull(email)) {
+            throw new IllegalArgumentException("Name and email of user requesting support is required");
+        }
+        if (Objects.isNull(subject)) {
```

Comment:
I carried these null checks over from `SupportRequestService.createSupportTicket`, but most existing model classes don't check for null values in the constructor. Thoughts on whether to keep or remove?

Reply:
I think we should keep these here, they are all required fields.